### ISSUE TITLE
Enable cloud tracing in e2e framework

### ISSUE THREAD

**Issue by sebastienvas** [istio/istio#241]
The framework should gather trace logs and upload them with other logs.

Cluster are set to use stackdriver. We need to create a sync before starting the tests and export the logs at the end of the end. At the same time we should gather information about kubenetes deployments (pod, services, ingress, etc..) to help debugging.

**Comment by rshriram** [comment]
Where are these logs being uploaded?

**Comment by sebastienvas** [comment]
Here is an example
http://gcsweb.istio.io/gcs/istio-artifacts/1ce783488cb0af012dfe6cb559f41402433bde40

**Comment by kyessenov** [comment]
Are you extracting logs from Stackdriver Tracing? One suggestion: don't
split logs by INFO/WARNING/ERROR. It would be nice to have logs for each
proxy (sidecar, ingress, egress), manager (discovery, apiserver), and mixer.
